qir/qat/Rules/Notation/Notation.cpp.orig

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Rules/Notation/Notation.hpp"
+#include "Rules/Patterns/AnyPattern.hpp"
+#include "Rules/Patterns/CallPattern.hpp"
+#include "Rules/Patterns/Instruction.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <unordered_map>
+#include <vector>
+
+namespace microsoft
+{
+namespace quantum
+{
+    namespace notation
+    {
+
+        ReplacerFunction deleteInstruction()
+        {
+            return [](ReplacementRule::Builder&, ReplacementRule::Value* val, ReplacementRule::Captures&,
+<<<<<<< HEAD
+                      ReplacementRule::Replacements& replacements)
+            {
+                auto type = val->getType();
+=======
+                      ReplacementRule::Replacements& replacements) {
+                auto type  = val->getType();
+                auto instr = llvm::dyn_cast<llvm::Instruction>(val);
+
+                if (instr == nullptr)
+                {
+                    return false;
+                }
+
+>>>>>>> main
```

Comment:
What is this file for? Forgot to remove? Otherwise there is an unresolved merge conflict.

Reply:
Yes, it is trash.